pages/common/acme.sh.md

```diff
@@ -0,0 +1,36 @@
+# acme.sh
+
+> An ACME Shell script, an acme client alternative to certbot.
```

Comment:
This is more verbatim from the repository.
```suggestion
> Shell script implementing ACME client protocol, alternative to certbot.
```

Reply:
Nice, also what about *an* alternative to certbot?

Reply:
Updated 👍🏻